themes/snow.js

```diff
@@ -70,7 +70,7 @@ class SnowTooltip extends BaseTooltip {
   }
 }
 SnowTooltip.TEMPLATE = [
-  '<a class="ql-preview" target="_blank" href="about:blank"></a>',
+  '<a class="ql-preview" rel="noopener noreferrer nofollow" target="_blank" href="about:blank"></a>',
```

Comment:
`nofollow` does not seem relevant to tabnabbing?

Reply:
Fair, `noopener` is sufficient to mitigate tabnabbing. I've left in noreferrer for privacy reasons but can remove that too if you think it's too much.

Reply:
That's fine these two are what React's linter suggests as well